opflexagent/vif_plug_vpp/linux_net.py

```diff
@@ -0,0 +1,112 @@
+# Derived from nova/network/linux_net.py
+#
+# Copyright (c) 2011 X.commerce, a business unit of eBay Inc.
+# Copyright 2010 United States Government as represented by the
```

Comment:
Make the licensing consistent with other files

Reply:
Will do..